force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -94,7 +123,18 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the input is "Empty", "Null", or "Contains Text".
      */
     public static String checkString(String a) {
-        return null; // Replace null with the variable you used to store the result
+        // Check if the input string is null
+        if (a == null) {
+            return 'Null';
+        }
+    
+        // Check if the input string is empty (i.e., has zero length)
+        if (String.isEmpty(a)) {
```

Comment:
String.isEmpty() also catches null. Which in this case is fine, because you are already checking for null with a return above. Just an FYI for future usage.